src/main/java/edu/kit/provideq/toolbox/api/SolveRouter.java

```
@@ -142,6 +135,146 @@ private Mono<ServerResponse> handleSolutionRouteForMetaSolver(MetaSolver<?, ?, ?
+              solution.abort();
+            });
+  }
+
+  private static org.springdoc.core.fn.builders.exampleobject.Builder getExampleOk(
+          String exampleName,
+          MetaSolver<?, ?, ?> metaSolver,
+          Consumer<Solution<String>> solutionModifier) {
+    // Prepare a solved solution with some example data
+    var solvedSolution = new Solution<String>(42);
+    solvedSolution.setExecutionMilliseconds(42);
+    metaSolver.getAllSolvers().stream()
+            .findFirst()
+            .ifPresent(solver -> solvedSolution.setSolverName(solver.getName()));
+    solutionModifier.accept(solvedSolution);
+
+    // Convert the solution to a string
+    String solvedSolutionString;
+    try {
+      solvedSolutionString = new ObjectMapper().writeValueAsString(solvedSolution);
+    } catch (JsonProcessingException e) {
+      solvedSolutionString = "Error: example could not be parsed";
+    }
+
+    // Build the example
+    return org.springdoc.core.fn.builders.exampleobject.Builder
+            .exampleOjectBuilder()
+            .name(exampleName)
+            .description("The problem was solved successfully.")
+            .value(solvedSolutionString);
+  }
+
+  private static org.springdoc.core.fn.builders.apiresponse.Builder getResponseNotFound() {
+    return responseBuilder()
+            .responseCode(String.valueOf(HttpStatus.NOT_FOUND.value()));
+  }
+
+  private org.springdoc.core.fn.builders.content.Builder getRequestContent(
+          MetaSolver<?, ?, ?> metaSolver) {
+    String content = metaSolver.getExampleProblems()
+            .stream().findFirst()
+            .map(e -> {
+              if (e instanceof String) {
+                return (String) e;
+              }
+
+              try {
+                return new ObjectMapper().writeValueAsString(e);
+              } catch (JsonProcessingException exception) {
+                return "Error: example could not be parsed";
```

please throw an error instead